ui/components/multichain/pages/send/send.test.js

```diff
@@ -36,6 +36,17 @@ describe('SendPage', () => {
   describe('render', () => {
     it('renders correctly', () => {
       const { container, getByTestId } = render();
+      ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
```

Comment:
This fencing is unnecessary and will not change anything. Fencing is not run in Jest.